app/views/pages/about.html.erb

```diff
@@ -5,15 +5,15 @@
 <main role="main" id="content">
   <%= render partial: 'layouts/feedback_survey_banner' if flash[:beta_message] == 'hide'%>
 
-  <h1 class="heading-large">
+  <h1 class="heading-xlarge">
```

Comment:
Not sure we want to change those classes either. That would break the design. Those classes exist to separate the heading level (h1-h6) from its appearance. The levels have to be a proper hierarchy, while the classes are "free"

Reply:
I thought the classes are tied to the headings - as at https://govuk-elements.herokuapp.com/typography/#typography-headings but happy not to